Gruntfile.coffee

```diff
@@ -248,8 +248,9 @@ module.exports = (grunt) ->
       fonts:
         files: [
           {src: ["bootstrap/*"], dest: "build/fonts", cwd: "bower_components/bootstrap-sass/assets/fonts", expand: true}
-          {src: ["*"], dest: "build/fonts", cwd: "assets/fonts/bc-icons", expand: true}
-          {src: ["*"], dest: "build/fonts", cwd: "assets/fonts/roboto", expand: true}
+          {src: ["*"], dest: "build/fonts", cwd: "assets/fonts/montserrat", expand: true}
+          {src: ["*"], dest: "build/fonts", cwd: "assets/fonts/gillsans", expand: true}
+          {src: ["*"], dest: "build/fonts", cwd: "assets/fonts/icomoon", expand: true}
```

Comment:
Thanks. I tested `make dist` and it looks good.

Can you consider using Bower or NPM to pull in the new fonts, rather than adding them into the repo?